### PR TITLE
feat: add amount tradable and retired to send confirmation modal

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#961

- add amount tradable and retired to send confirmation modal in `useCreditSendSubmit.tsx` hook
- fix typo on `useUpdateCreditsBaskets` hook name

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. Go to https://deploy-preview-996--regen-registry.netlify.app/ecocredits/dashboard
2. Send a credit (optionally using retire)
3. Check that tradable amount is displayed (and retired if applicable)

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/pages/MyEcocredits/hooks/useUpdateCreditsBaskets.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ type Props = {
 
 type ReturnType = void;
 
-const useUpdateCrediBaskets = ({
+const useUpdateCreditsBaskets = ({
```
Comment (flagrede) [inline web-registry/src/pages/MyEcocredits/hooks/useUpdateCreditsBaskets.tsx:14]: fix typo

File: web-registry/src/pages/MyEcocredits/hooks/useCreditSendSubmit.tsx [hunk 2]
```diff
@@ -55,16 +55,26 @@ const useCreditSendSubmit = ({
...
+              value: { name: batchDenom, url: `/credit-batches/${batchDenom}` },
+            },
+            {
+              label: 'amount tradable',
+              value: { name: values.tradableAmount.toString() },
+            },
+            {
+              label: 'amount retired',
+              value: { name: values.retiredAmount.toString() },
+            },
```
Comment (flagrede) [inline web-registry/src/pages/MyEcocredits/hooks/useCreditSendSubmit.tsx:71]: new fields

**Comment by flagrede** [comment]
@erikalogie I added steps to test this issue.

**Comment by erikalogie** [comment]
Looks great!

**Review by mhagel — APPROVED** [review]
One comment. Pre-approving. Thanks!

**Review by haveanicedavid — APPROVED** [review]

**Comment by flagrede** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**